cookiecutter-django-ida/{{cookiecutter.repo_name}}/Makefile

```diff
@@ -32,6 +32,10 @@ clean: ## delete generated byte code and coverage reports
 	rm -rf assets
 	rm -rf pii_report
 
+docs: ## generate Sphinx HTML documentation, including API docs
+	tox -e docs
+	$(BROWSER)docs/_build/html/index.html
```

Comment:
[nit] should there be a space here after $(BROWSER)?

Reply:
Interestingly enough, no! ...but only if the correct `BROWSER` is defined above, which was not the case. There's a different, more complicated version used in about 40% of our Makefiles that does require a space. However, the other Makefiles in this repo use the short, no-space version, so I'll switch it to that. Good catch!